.cicd/platforms/pinned/macos-10.15-pinned.sh

```diff
@@ -4,6 +4,8 @@ VERSION=1
 export SDKROOT="$(xcrun --sdk macosx --show-sdk-path)"
 brew update
 brew install git cmake python libtool libusb graphviz automake wget gmp pkgconfig doxygen openssl jq rabbitmq postgres || :
+# install request and requests_unixsocket module
+pip3 install requests && pip3 install requests_unixsocket
```

Comment:
This can just be
`pip3 install requests requests_unixsocket`

Reply:
Done.